files/image_config/interfaces/interfaces.j2

```diff
@@ -43,58 +43,61 @@ iface eth0 inet dhcp
 {% endif %}
 #
 {% endblock mgmt_interface %}
+
 {% block front_panel_interfaces %}
 # The switch front panel interfaces
+{% for port in minigraph_ports.keys()|sort %}
+auto {{ port }}
+allow-hotplug {{ port }}
+{% endfor %}
+
 {% for interface in minigraph_interfaces %}
-auto {{ interface['alias'] }}
-allow-hotplug {{ interface['alias'] }}
-iface {{ interface['alias'] }} {{ 'inet' if interface['addr'] | ipv4 else 'inet6' }} static
+iface {{ interface['attachto'] }} {{ 'inet' if interface['addr'] | ipv4 else 'inet6' }} static
     address {{ interface['addr'] }}
     netmask {{ interface['mask'] }}
-#
 {% endfor %}
-{% for vlan_interface in minigraph_vlan_interfaces|unique_name %}
-{% for interface in vlan_interface['members'] %}
-auto {{ interface }}
-allow-hotplug {{ interface }}
-iface {{ interface }} inet manual
-    pre-up ifconfig {{ interface }} up
-    post-up brctl addif {{ vlan_interface['name'] }} {{ interface }}
-    post-down ifconfig {{ interface }} down
-#
+{% for vlan in minigraph_vlans.keys()|sort %}
+{% for member in minigraph_vlans[vlan]['members'] %}
+iface {{ member }} inet manual
+    pre-up ifconfig {{ member }} up
+    post-up brctl addif {{ vlan }} {{ member }}
+    post-down ifconfig {{ member }} down
 {% endfor %}
 {% endfor %}
-# Add || true to suppress the error when docker-teamd starts after docker-swss
-{% for pc_interface in minigraph_portchannel_interfaces|unique_name %}
-{% for interface in pc_interface['members'] %}
-{% if pc_interface['name'] not in pc_set %}
-auto {{ interface }}
-allow-hotplug {{ interface }}
-iface {{ interface }} inet manual
-    pre-up teamdctl {{ pc_interface['name'] }} port add {{ interface }} || true
-    post-down ifconfig {{ interface }} down
-#
-{% endif %}
+# "|| true" is added to suppress the error when docker-teamd starts after docker-swss
+{% for pc in minigraph_portchannels.keys()|sort %}
+{% for member in minigraph_portchannels[pc]['members'] %}
+iface {{ member }} inet manual
+    pre-up teamdctl {{ pc }} port add {{ member }} || true
+    post-down ifconfig {{ member }} down
 {% endfor %}
 {% endfor %}
 {% endblock front_panel_interfaces %}
 {% block vlan_interfaces %}
+
+# Vlan interfaces
+{% for vlan in minigraph_vlans.keys()|sort %}
+auto {{ vlan }}
+{% endfor %}
+
```

Comment:
it turns out to be that bridge_ports should follow iface line. thanks @taoyl-ms . let's stick auto stanza with iface stanza together for better readability.